cmd/promxy/main.go

```diff
@@ -154,10 +155,16 @@ func main() {
 	}
 	logrus.SetLevel(level)
 
-	// Set the log format to have a reasonable timestamp
-	formatter := &logrus.TextFormatter{
-		FullTimestamp: true,
+	var formatter logrus.Formatter
+	if opts.LogFormat == "json" {
+		formatter = &logrus.JSONFormatter{}
```

Comment:
This has some options for timestamp format, is the default okay? (doesn't specify in the docs which it is)

Reply:
TextFormat FullTimestamp: true = 2020-07-25T13:47:44Z
JSONFormatter = "time":"2020-07-25T10:58:20Z"
the default is okay :)